src/test/java/edu/oakland/test/display01/DisplayGpsManagerStub.java

```diff
@@ -0,0 +1,35 @@
+package edu.oakland.test.display01;
+
+
+import edu.oakland.helper.admin.Satellite;
+import edu.oakland.helper.display01.SatelliteSignalCheckRequest;
+import edu.oakland.production.display01.DisplayGpsManager;
+
+
+public class DisplayGpsManagerStub implements DisplayGpsManager {
+
+  private String checkType = "oreo";
+  
+  /**
+   * Receives Gps Signal Strength and pulls satellite name.
+   *
+   * @param satelliteSignal passes satellite object to pull name and checktype from
+   */
+  public String receiveGpsSignalStrength(Satellite satelliteSignal) {
+    SatelliteSignalCheckRequest ssrc = 
+        new SatelliteSignalCheckRequest(satelliteSignal.getSatelliteName(), checkType);
+    return ssrc.getSatelliteName();
+  }
+
+  /**
+   * Receives Gps Signal Strength and pulls satellite name.
+   *
+   * @param satelliteSignal passes satellite object to pull name and checktype from
+   */
+  public SatelliteSignalCheckRequest passGpsSignalStrength(Satellite satelliteSignal) {
+    SatelliteSignalCheckRequest ssrc = 
+        new SatelliteSignalCheckRequest(satelliteSignal.getSatelliteName(), checkType);
```

Comment:
Just a little cleaner formatting:
```suggestion
    SatelliteSignalCheckRequest ssrc = new SatelliteSignalCheckRequest(
        satelliteSignal.getSatelliteName(),
        checkType
    );
```